build/tools/chainlink-fulfilling-requests.md

```diff
@@ -0,0 +1,161 @@
+
+Confirm the transaction in Metamask and record the address of your Oracle as we will need it to create a job for our node.
+
+## Creating jobs
+
+Note: Your node must be actively running. You can re-establish a connection to your active node by running the command:
+
+```bash
+ssh -i $KEY $USER@$REMOTE-IP -L 6688:localhost:6688 -N
+```
+
+If you still can't connect, please refer to the [node deployment article](chainlink-node-deployment.md).
+
+For this tutorial we will use the [EthUnit256 job specification](https://docs.chain.link/docs/fulfilling-requests/). Other Job specifications can be found [here!](https://docs.chain.link/docs/job-specifications/)
+
+Open the Chainlink GUI, which should be at [http://localhost:6688/](http://localhost:6688/) in your web browser and navigate to the **Jobs** tab. There, paste the following into the JSON section with your oracle's contract address:
+
+```json
+{
+  "name": "Get > Uint256",
+  "initiators": [
+    {
+      "type": "runlog",
+      "params": {
+        "address": "YOUR_ORACLE_CONTRACT_ADDRESS"
+      }
+    }
+  ],
+  "tasks": [
+    {
+      "type": "httpget"
+    },
+    {
+      "type": "jsonparse"
+    },
+    {
+      "type": "multiply"
+    },
+    {
+      "type": "ethuint256"
+    },
+    {
+      "type": "ethtx"
+    }
+  ]
+}
+```
+You can also name your job. Click the **Create Job** button, and you should see a confirmation at the top of your screen!
+
+![Job confirmation](../../.github/chainlinkp2-06-Screen_Shot_2021-07-18_at_1.31.17_PM_2.png)
+
+Now our node can begin fulfilling requests!
+
+## Fulfill a request
+
+Note: Changed currentPrice, price and, _price from bytes32 to Uint256
+
+View the **Consumer.sol** file and see line 21
+
+```jsx
+req.add("get", "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD,EUR,JPY");
+```
+
+We will use Chainlink's "**get**" request on the end point "[**https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD,EUR,JPY**](https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD,EUR,JPY)" to get ETH/USD price data.
+
+End points and data parameters can be modified using "paths". Please see [Job Specifications](https://docs.chain.link/docs/job-specifications/)  and [Fulfilling Requests](https://docs.chain.link/docs/fulfilling-requests/) for more info.
+
+We must convert our jobSpecId into Bytes32 for our contact to read it. Navigate to your created job and then click on the **Definition** tab, find your jobSpecId and convert it into bytes32 using a tool of your choice. I recommend using [https://web3-type-converter.onbrn.com/](https://web3-type-converter.onbrn.com/). Be sure to use the jobSpecID **without** dashes. In this example case, it looks like this: `95b514ab9b6449b1aa55f389ae517b5a`.
+
+![SpecID](../../.github/chainlinkp2-07-Screen_Shot_2021-07-18_at_1.40.49_PM_2.png)
+
+Select and compile the **BasicConsumer.sol** file. Enter the LINK, ORACLE, and SPECID parameters. Press `transact` to deploy!
+
```

Comment:
Compilation and deployment are not entirely clear, some more guidance could be helpful.